pkg/util/exec-run.go

```diff
@@ -0,0 +1,43 @@
+package util
+
+import (
+	"io/ioutil"
+	"os/exec"
+
+	"github.com/golang/glog"
+)
+
+type Runner interface {
+	RunCombinedOutput(string, ...string) ([]byte, error)
+	RunStdoutPipe(string, ...string) ([]byte, error)
+}
+
+type RealRunner struct{}
+
+// the real runner for the actual program, actually execs the command
+func (r RealRunner) RunCombinedOutput(command string, args ...string) ([]byte, error) {
+	//execute pool creation command.
+	cmd := exec.Command(command, args...)
+	out, err := cmd.CombinedOutput()
+	return out, err
+}
+
+func (r RealRunner) RunStdoutPipe(command string, args ...string) ([]byte, error) {
```

Comment:
can we give one example of terminal command(s) where RunStdoutPipe is useful/should be used

Reply:
same as above.